src/main/java/uk/gov/hmcts/reform/civil/notification/handlers/createclaimspecafterpayment/ClaimContinuingOnlineSpecRespSolOneEmailDTOGenerator.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.hmcts.reform.civil.notification.handlers.createclaimspecafterpayment;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.notification.handlers.RespSolOneEmailDTOGenerator;
+import uk.gov.hmcts.reform.civil.notify.NotificationsProperties;
+import uk.gov.hmcts.reform.civil.prd.model.Organisation;
+import uk.gov.hmcts.reform.civil.service.OrganisationService;
+
+import java.util.Map;
+import java.util.Optional;
+
+import static uk.gov.hmcts.reform.civil.helpers.DateFormatHelper.DATE;
+import static uk.gov.hmcts.reform.civil.helpers.DateFormatHelper.formatLocalDate;
+
+@Component
+public class ClaimContinuingOnlineSpecRespSolOneEmailDTOGenerator extends RespSolOneEmailDTOGenerator {
+
+    private static final String REFERENCE_TEMPLATE = "claim-continuing-online-notification-%s";
+
+    private final NotificationsProperties notificationsProperties;
+
+    public ClaimContinuingOnlineSpecRespSolOneEmailDTOGenerator(
+            NotificationsProperties notificationsProperties,
+            OrganisationService organisationService
+    ) {
+        super(organisationService);
+        this.notificationsProperties = notificationsProperties;
+    }
+
+    @Override
+    protected String getEmailTemplateId(CaseData caseData) {
+        return notificationsProperties.getRespondentSolicitorClaimContinuingOnlineForSpec();
+    }
+
+    @Override
+    protected String getReferenceTemplate() {
+        return REFERENCE_TEMPLATE;
+    }
+
+    @Override
+    protected Map<String, String> addCustomProperties(Map<String, String> properties,
+                                                      CaseData caseData) {
+
+        properties.put(CLAIM_DEFENDANT_LEGAL_ORG_NAME_SPEC, getRespondentLegalOrganizationName(
```

Comment:
is it not same as getLegalOrganizationNameForRespondent(caseData,
            isRespondent1, organisationService)); which can be copied from RespSolOneEmailDTOGenerator